plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/PredicatePushdownController.java

```diff
@@ -30,7 +30,7 @@
         }
         return new DomainPushdownResult(domain, Domain.all(domain.getType()));
     };
-    PredicatePushdownController PUSHDOWN_AND_KEEP = (session, domain) -> new DomainPushdownResult(
+    PredicatePushdownController CASE_INSENSITIVE_PUSHDOWN = (session, domain) -> new DomainPushdownResult(
```

Comment:
Also, the old name (`PUSHDOWN_AND_KEEP`) is _correct_ -- it correctly describes what the thing does.
The new name is _aspirational_, but not _correct_. This commit should be squashed with `Fix CASE_INSENSITIVE_PUSHDOWN` commit. I would be OK separating this as a refactor, if it was a big and repetitive preparatory refactor, but this is just 3 lines being changed.